packages/dashboard-provider/test/MockDashboard.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  base64ToJson,
+  connectToMessageBusWithRetries,
+  isDashboardProviderMessage,
+  jsonToBase64,
+  Message
+} from "@truffle/dashboard-message-bus";
+import { JSONRPCRequestPayload } from "ethereum-protocol";
+import { promisify } from "util";
+import WebSocket from "ws";
+import Ganache from "ganache-core";
+
+// TODO: This mock dashboard was copy-pasted from the dashboard-provider tests
+// We should figure out whether we want to make this DRYer
+export default class MockDashboard {
```

Comment:
But... these are the `dashboard-provider` tests! 😉 

See my analogous comment over in `packages/dashboard/test/MockDashboard.ts` for more explanation, but could you replace the `TODO` with maybe this:

```suggestion
// NOTE This mock dashboard was copy-pasted from the dashboard tests
export default class MockDashboard {
```

Reply:
Good point 😆